src/amo/components/RatingManager/index.js

```diff
@@ -196,10 +196,13 @@ export class RatingManagerBase extends React.Component<InternalProps, State> {
   };
 
   render() {
-    const { i18n, addon, userId, userReview } = this.props;
+    const { i18n, addon, userId, userReview, version } = this.props;
     const { showTextEntry } = this.state;
     const isLoggedIn = Boolean(userId);
 
+    invariant(addon, 'addon is required');
+    invariant(version, 'version is required');
```

Comment:
I removed an if statement that was checking the existence of `addon`. It wasn't necessary because `addon` is required. However, `<Addon>` is the caller of `<RatingManager>` and that component isn't covered by Flow so I thought it would be a good idea to add some invariant protection. While I was at it, I added protection for `version` which is the only other required prop.